LocalSettings.php

```diff
@@ -4038,6 +4035,7 @@
 	],
 
 	// Restriction types
+	// Per wmfphab:T230103, they should ideally follow the format of editXXprotected
```

Comment:
We aren't Wikimedia, and this does not have to be a standard on Miraheze if it isn't how individual wikis want them. The comment is probably unnecessary.